samza-core/src/main/scala/org/apache/samza/config/StorageConfig.scala

```diff
@@ -51,7 +50,7 @@ class StorageConfig(config: Config) extends ScalaMapConfig(config) with Logging
   def getStorageMsgSerde(name: String) = getOption(StorageConfig.MSG_SERDE format name)
 
   def getAccessLogEnabled(storeName: String) = {
-    getBoolean(ACCESSLOG_ENABLED format storeName, false)
+    new JavaStorageConfig(config).getAccessLogEnabled(storeName);
```

Comment:
Minor: Don't need `;` in scala, here and elsewhere.

Reply:
Good catch.